x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/parser/ExpressionTests.java

```diff
@@ -136,6 +136,19 @@ public void testBackQuotedAttribute() {
         assertThat(ua.qualifier(), is(nullValue()));
     }
 
+    public void testBackQuotedIdentifierWithEscapedBackQuote() {
```

Comment:
Please add some tests for failures, e.g.:
- backquote unescaped
- double backquotes
- escaped backslash and backquote: ``\\\` ``